### ISSUE TITLE
Source for v1.10.0 is broken

### ISSUE THREAD

**Issue by braderhart** [gopasspw/gopass#1548]
### Summary

Hello, the official Arch package which compiles from the archived source here is broken and still references version 1.10.0-rc.1:

https://github.com/archlinux/svntogit-community/blob/packages/gopass/trunk/PKGBUILD

Additionally, `gopass-jsonapi` and the other tools are broken as well and simply run the compiled `gopass` binary. They appear to be working in the compiled releases.

**Comment by petobens** [comment]
Hi! Is this the same cause of the following error that I've started seeing today?
```
$> gopass --version
Failed to load config *config.Config. unknown fields: [root]
gopass 1.10.0-rc.1 (2020-08-23 18:46:22) go1.15 linux amd64
```

**Comment by braderhart** [comment]
@petobens Yes, that is the same cause.

**Comment by dominikschulz** [comment]
The error message is actually only a warning. We tried to be more helpful, but actually this might be more confusing than helpful.

The `Makefile` is just broken for the external binaries.
Our binary releases don't rely on the Makefile so these should be fine.
But anything using the Makefile will be affected.

**Comment by braderhart** [comment]
Is there any plan to fix the Makefile in the near future? What do your binary releases rely on? Can you share the build instructions? Thanks!

**Comment by r3k2** [comment]
yes please fix the makefile is bad systems engineering to install things from source(out of OS packaging) unless you are going to develop on the app or there is a new feature that you need.

**Comment by dominikschulz** [comment]
There is a PR to fix the Makefile: #1550
Does anyone want to review it?
